recipes/nm.rb

```diff
@@ -188,3 +230,18 @@
   end
 end
 
+
+template "/etc/sudoers.d/yarn" do
```

Comment:
@jimdowling Could you also put this sudoers in within the `if` above?

Reply:
i put it inside the "if cloud == true" || localhost == true" condition. i assume that's where you meant.